server/team/team.route.js

```diff
@@ -8,9 +8,17 @@ const router = express.Router(); // eslint-disable-line new-cap
 router
   .route('/')
   /** GET /api/teams - Get list of users */
-  // .get(teamCtrl.list)
-
+  .get(teamCtrl.list)
   /** POST /api/teams - Create new team */
   .post(ifNoUserRedirect(), teamCtrl.create);
 
+router
+  .route('/analyze')
+  .get(teamCtrl.analyze);
+
+router
+  .route('/join')
+  .get(teamCtrl.joinForm)
```

Comment:
This highlights something funky about the boilerplate I used for this. The route for this will be `/api/teams/join`. the `/api/*` route should not be used for rendering pages.

It is out of the scope of this PR, but I want to create a view controller that handles all the view routing and mount it at '/'.